Update create-rules.mdx (#17339)
cloudflare/cloudflare-docs: src/content/docs/load-balancing/additional-options/load-balancing-rules/create-rules.mdx

To save a new load balancer rule, make sure to save both the rule **and** the overall load balancer configuration.

:::

:::note

> 

Reviewer: ```suggestion
```

> Generally speaking, for non-terminating actions, the last change made by rules in the same [phase](/ruleset-engine/about/phases/) will win (later rules can overwrite changes done by previous rules). However, for [terminating actions](/ruleset-engine/rules-language/actions/) (Block, Redirect, or one of the challenge actions), rule evaluation will stop and the action will be executed immediately. 

Reviewer: ```suggestion
In general, for non-terminating actions, the last change made by rules within the same [phase](/ruleset-engine/about/phases/) will win (later rules can overwrite changes made by previous ones). However, for [terminating actions](/ruleset-engine/rules-language/actions/), such as Block, Redirect, or any of the challenge actions, rule evaluation will stop and the action is executed immediately.
```

> Load Calancer Custom Rules overrides the default Load Balancer setting including pool/origin selection. These are non-terminating actions so the last rule will override the previous rules.

Reviewer: ```suggestion
Load Balancer Custom Rules override the default Load Balancer settings, including pool and origin selection. These are non-terminating actions, so the last rule applied will override any prior rules.
```
